handbook/community.md

```diff
@@ -190,37 +190,22 @@ graph TD;
 
 ## Community contributions (pull requests)
 
-The top priority when community members contribute PRs is to help the person feel engaged with
-Fleet. This means acknowledging the contribution quickly (within 1 business day) and driving to a
-resolution (close/merge) as soon as possible (may take longer than 1 business day).
+When community members contribute PRs Fleet’s top priority is to help them feel engaged. We must acknowledge contributions as soon as possible, within 1 business day. We must also drive to a resolution (close/merge) as soon as possible, though this may take longer than 1 business day.
```

Comment:
> When community members contribute changes, Fleet's…

- PRs => changes
- add comma

Reply:
> We must acknowledge contributions
We acknowledge contributions…

> We must also drive to a resolution
We also drive to a resolution (close/merge) as soon as possible…